CONTRIBUTING.md

```diff
@@ -94,14 +94,25 @@ long-term, portability, and the scope of the impact on the code base. Therefore,
 Modeling Systems does not guarantee that all pull requests will be accepted,
 even if the changes pass the initial testing phases, and are otherwise correct.
 
+## Commits
+
+FMS uses the guidelines specified by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).
+
+FMS pull requests are typically ‘squashed’ into one commit.
+Descriptive commit messages within PR’s should still be used, and some projects / developers may find it useful to include the file and routine being updated for easy reference.
+The commit message on the main branch must follow the guidelines of conventional commits, meaning it includes a ‘type’ prefix (ie. usually fix or feat(ure)) and a footer for any API-breaking changes.
+Additionally, commits should include the number of the pull request in parentheses (this is automatically added on github) to allow easier tracking of each commit.
+
 ## Tests
 
-FMS uses TravisCI and gitlab-CI to run build tests for libFMS.  Users may create
+FMS uses github actions workflows to run build, runtime, and code linting tests for libFMS.  Users may create
```

Comment:
Is it _may_ or _must_?

Reply:
Perhaps "_may be required to_" because it really depends on the contribution.  If something already has a test, then the developer won't be required to add one.